ci/tests/e2e-helm-scaffold-hybrid.sh

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env bash
+
+source hack/lib/test_lib.sh
+
+set -eux
+
+ROOTDIR="$(pwd)"
+HELMDIR="/go/src/github.com/helm-op"
+mkdir -p $HELMDIR
+
+# create and build the operator
+pushd "$HELMDIR"
+log=$(operator-sdk new nginx-operator --api-version=helm.example.com/v1alpha1 --kind=Nginx --type=helm --no-helm-role 2>&1)
+echo $log
+if echo $log | grep -q "failed to generate RBAC rules"; then
+    echo FAIL expected successful generation of RBAC rules
+    exit 1
+fi
```

Comment:
These tests were mainly meant to verify the cluster-based role generation. So without a cluster, I think its probably okay to remove them, which also helps us avoid adding the `--no-helm-role` flag.

We also now have unit tests for the role generation [here](https://github.com/operator-framework/operator-sdk/blob/master/internal/pkg/scaffold/helm/role_test.go), which uses a mock discovery client to avoid the cluster requirement, so we should still have reasonable test coverage.

Reply:
I think when I first worked on this, the `operator-sdk new` command was failing for helm operators if there was no available cluster, but it seems like that's not happening anymore, so I can remove that

Reply:
Actually, the problem is that it fails if it can't find a kubeconfig

Reply:
Huh, someone just made a new issue about this as well: https://github.com/operator-framework/operator-sdk/issues/1625